### PR TITLE
Release images on PR merge and tags

### PR DESCRIPTION
**Pull request by slintes**
- added relevant git action
- updated Makefile with correct image repository
- fix typo in CSV
- run make bundle
- update README with new API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +22,10 @@ In order to set up: make sure that Node Health Check is running, Machine-deletio
 ## Example CRs
 An example MDR template object.
 ```yaml
-   apiVersion: machine.remediation.medik8s.io/v1alpha1
-   kind: MachineDeletionRemediationTemplate
+   apiVersion: machine-deletion.medik8s.io/v1alpha1
```
Comment (clobrano) [inline README.md:25]: this should be machine-deletion-remediation, I think. It was already changed in a previous PR.
The same goes for similar changes below.

File: bundle.Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=machine-deletion
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.7.1+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.11.0+git
```
Comment (clobrano) [inline bundle.Dockerfile:9]: Currently it is v1.25.1, isn't Ok?
https://github.com/medik8s/machine-deletion-remediation/blob/c14de13b378c08056c4b0cb5a9d1f1c249ffcddb/bundle.Dockerfile#L10

File: config/manifests/bases/machine-deletion.clusterserviceversion.yaml [hunk 3]
```diff
@@ -33,7 +32,7 @@ spec:
     strategy: ""
   installModes:
   - supported: false
-    type: OwnNamespacell
+    type: OwnNamespace
```
Comment (clobrano) [inline config/manifests/bases/machine-deletion.clusterserviceversion.yaml:35]: This is surely VIM to blame :smile: 
thanks for catching it

**Comment by clobrano** [comment]
nevermind, I forgot this was opened in 2022 :facepalm:

**Comment by slintes** [comment]
> nevermind, I forgot this was opened in 2022 facepalm

Still great to finally get a review 😎 
Do you want me to rebase, or do you prefer to add this feature with a new PR on your own?

**Comment by clobrano** [comment]
> > nevermind, I forgot this was opened in 2022 facepalm
> 
> Still great to finally get a review sunglasses Do you want me to rebase, or do you prefer to add this feature with a new PR on your own?

I think it's easier for me to pick the changes that are still valid, thank you

**Pull request closed without merging**